CMakeLists.txt

```diff
@@ -75,6 +75,11 @@ option(FLATBUFFERS_SKIP_MONSTER_EXTRA
       "Skip generating monster_extra.fbs that contains non-supported numerical\"
       types." OFF)
 
+set(MSVC_LIKE OFF)
```

Comment:
Can you define this inline without the explicit if statement

Reply:
I don't seem to find a way to inline the initialization for a "boolean" variable  in CMake. The syntax for `if(condition)` in CMake is special, so the `condition` here is not an expression that can be used in set.

I am not sure if I understand your comment correctly. If not, can you give an explicit edit suggestion? Thanks!